modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -27,12 +27,17 @@ The `ImageContentSourcePolicy` resource associates the mirror registry with the
 $ oc apply -f ./oc-mirror-workspace/results-1639608409/
 ----
 
-. Apply the release image signatures to the cluster by running the following command:
+. If you mirrored release images, apply the release image signatures to the cluster by running the following command:
 +
 [source,terminal]
 ----
 $ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/
 ----
++
+[IMPORTANT]
```

Comment:
Since you changed the wording in line 30, this admonition might not be necessary anymore, or maybe this could be downgraded to a NOTE since by this point users are already made aware of how this command can be skipped if they're not mirroring release images.